### PR TITLE
Animate Landing Page Lights

### PR DESCRIPTION
**Pull request by saidam90**
solves #504 

Animated lights, increased the size of lights (x3) and stretched the animation to 12s.

### CODE REVIEW

**Review by dcts — CHANGES_REQUESTED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -34,6 +32,26 @@ module.exports = {
...
+          "0%, 100%": { transform: "translate(40px, -40px) scale(3.2)" },
+          "20%": { transform: "translate(-30px, -50px) scale(3.1)" },
+          "40%": { transform: "translate(30px, -50px) scale(3)" },
+          "60%": { transform: "translate(-30px, 10px) scale(3.3)" },
+          "80%": { transform: "translate(30px, 10px) scale(3.2)" },
+        },
+      },
+      animation: {
+        blueLight: "blueLight 12s ease-in-out infinite",
+        greenLight: "breenLight 12s ease-in-out infinite",
```
[the remark this answers is not in the document]
Comment (saidam90) [reply tailwind.config.js]: I fixed the typo and pushed the 12s version again, but in any case I will upload the 9s video on discord for comparison

File: src/app/page.tsx [hunk 2]
```diff
@@ -3,12 +3,13 @@
 import { DexterButton } from "components/DexterButton";
 import StatsWidget from "components/StatsWidget";
 import { useTranslations, useHydrationErrorFix } from "hooks";
+// import * as tailwindConfig from "../../../tailwind.config";
```
Comment (dcts) [inline src/app/page.tsx:6]: remove outcommented line

File: src/app/page.tsx [hunk 3]
```diff
@@ -3,12 +3,13 @@
...
 
 enum Device {
   MOBILE = "MOBILE",
   DESKTOP = "DESKTOP",
 }
-
+//
```
Comment (dcts) [inline src/app/page.tsx:12]: remove empty comment

File: tailwind.config.js [hunk 4]
```diff
@@ -19,8 +19,6 @@ module.exports = {
       fontFamily: {
         inter: ["inter"],
       },
-      // stretch the grid so that footer is always at the bottom
-      // even on pages with little content
```
Comment (dcts) [inline tailwind.config.js:23]: Why was this comment removed? 🤔

**Review by dcts — CHANGES_REQUESTED** [review]

**Comment by dcts** [comment]
@saidam90 Looks great now, happy to merge once comments get resolved! 💯

**Review by dcts — APPROVED** [review]

File: tailwind.config.js (hunk 4 above)
Comment (dcts) [reply tailwind.config.js]: @saidam90 if you think this comment should be removed, please open a new PR with a reason why it should be removed :)

**Comment by dcts** [comment]
Just realized since the requested changes are so minor I can implement and commit directly.

**Comment by dcts** [comment]
@fliebenberg can you approve + merge this PR? I did, but since I made a small commit (just removing comments), I believe another maintainer review is required.

**Review by Radstakes — APPROVED** [review]

**Pull request merged**